keps/sig-network/1669-proxy-terminating-endpoints/README.md

```diff
@@ -269,7 +270,7 @@ regardless of their termination state. If this is undesired, workloads should be
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 - [X] Metrics
-  - Metric name: TBD
+  - Metric name: `sync_proxy_rules_no_endpoints_total`
```

Comment:
Hmm - I was looking for this metric definition in k/k and I can't found it. Does this already exist?

Reply:
Sorry this is a typo. It's `sync_proxy_rules_no_local_endpoints_total` https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/metrics/metrics.go#L152-L162